AUDIO/DisablePedPainAudio.md

```diff
@@ -5,11 +5,11 @@ ns: AUDIO
 
 ```c
 // 0xA9A41C1E940FB0E8 0x3B8E2D5F
-void DISABLE_PED_PAIN_AUDIO(Ped ped, BOOL toggle);
+void DISABLE_PED_PAIN_AUDIO(Ped ped, BOOL shouldDisable);
 ```
 
```

Comment:
Description can be added - `Disable ped pain audio`

Reply:
The description would be redundant.